utils/Makefile

```diff
@@ -35,29 +35,31 @@ clean:
 	@rm -fR build dist rpm-build MANIFEST htmlcov .coverage cover ooinstall.egg-info oo-install
 	@rm -fR $(NAME)env
 
+viewcover:
+	xdg-open cover/index.html
+
 virtualenv:
 	@echo "#############################################"
 	@echo "# Creating a virtualenv"
 	@echo "#############################################"
 	virtualenv $(NAME)env
+	. $(NAME)env/bin/activate && pip install setuptools==17.1.1
```

Comment:
The version changes below (from previous commit) were using `setuptools.version` which isn't available in the setuptools version on RHEL so we now use `pkg_resources import parse_version`. I think we'll want the local tests to line up when we notice things like this.